src/Coduo/PHPHumanizer/String/Truncate.php

```diff
@@ -33,7 +33,7 @@ public function __construct($text, $charactersCount, $append = '')
 
     public function __toString()
     {
-        if ($this->charactersCount < 0 || strlen($this->text) <= $this->charactersCount) {
+        if ($this->charactersCount < 0 || strlen($this->text) <= ($this->charactersCount + mb_strlen($this->append))) {
```

Comment:
I just find out that we have more places like that, I gonna merge it now and prepare separated PR for switching to mb_* functions